quickstart/101-azure-virtual-desktop/defaults.tfvars

```diff
@@ -0,0 +1,20 @@
+# Customized the sample values below for your environment and either rename to terraform.tfvars or env.auto.tfvars
```

Comment:
Since, I believe your intent is for this file to be an example input values, please add any actual default values to the variables.tf file and move these tfvars files to an environments folder. Also should probably rename the file to dev.tfvars or something.

Reply:
Updates as you suggested. Thank you.